src/components/RandomBeer.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import {Link} from 'react-router-dom';
+import { useState, useEffect } from 'react';
+import axios from 'axios';
+import Header from './Header';
+
+function RandomBeer() {
+    const [randomBeer, setRandomBeer] = useState([]);
+
+    const getFromApi = async () => {
+  
+    try {
+  
+      const apiCall = await axios.get('https://ih-beers-api2.herokuapp.com/beers/random')
+  
+      setRandomBeer(apiCall.data)
+      
+    } catch (error) {
+      console.log(error)
+    } 
+  }
+  
+  useEffect(() => {
+    getFromApi()
+  }, [])
+
+
+  return (
+    <div>
+    <Header/>
+        <div>
+            {randomBeer.map((randomBeer) => {
+                return (
+                    <>
+                   <img src={randomBeer.image_url} alt="beers"/>
+                   <Link to={`/beer/${randomBeer._id}`}><p>{randomBeer.name}</p></Link>
```

Comment:
Since you have `<Route path='/onebeer/:beerId' element={<BeerDetails />} /> ` on App.js, this Link won't work because it's sending you to another route that doesn't exist. 
``<Link to={`/beer/${randomBeer._id}`}><p>{randomBeer.name}</p></Link>``

So here, you would need to pass the same as you have on App.js:

``<Link to={`/onebeer/${randomBeer._id}`}><p>{randomBeer.name}</p></Link>``